x-pack/plugins/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -340,6 +360,23 @@ export function ErrorSampleDetails({
         <SampleSummary error={error} />
       )}
 
+      {coPilot.isEnabled() && promptParams ? (
+        <>
+          <EuiFlexItem>
+            <CoPilotPrompt
+              coPilot={coPilot}
+              title={i18n.translate(
+                'xpack.apm.errorGroupDetails.explainErrorTitle',
+                { defaultMessage: "What's this error?" }
+              )}
+              promptId={CoPilotPromptId.ApmExplainError}
+              params={promptParams}
+            />
+          </EuiFlexItem>
+          <EuiSpacer size="s" />
+        </>
+      ) : undefined}
```

Comment:
Can you extract this to a separate component?

Reply:
It's a little bit complicated as I also have to render `TabContent` (which is a gross hack, apologies). I can export that though, then I won't have to render it every time even if Co-Pilot has not been enabled.

Reply:
Done, it's its own component now